synapse/storage/room.py

```diff
@@ -507,3 +507,27 @@ def get_ratelimit_for_user(self, user_id):
             ))
         else:
             defer.returnValue(None)
+
+    @cached(max_entries=10000)
+    def is_room_blocked(self, room_id):
+        return self._simple_select_one_onecol(
+            table="blocked_rooms",
+            keyvalues={
+                "room_id": room_id,
+            },
+            retcol="1",
```

Comment:
This is slightly evil, I suspect that people looking at the _simple_select_one_onecol wouldn't expect someone to pass "1" as a column ID. But it's probably fine for now. :(